xml/install_caasp_heat_templates.xml

```diff
@@ -116,7 +116,507 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
+    </screen>
+   </step>
+   <step>
+    <para>
+     Find the floating IP address of the load balancer. This is necessary for
+     accessing the Velum &caasp; dashboard.
+    </para>
+    <substeps>
+     <step>
+      <screen>&prompt.ardana;openstack loadbalancer list --provider</screen>
+     </step>
+     <step>
+      <para>
+       From the output, copy the <literal>id</literal> and enter it in the
+       following command as shown in the following example:
+      </para>
+      <screen>&prompt.ardana;openstack loadbalancer show id</screen>
+      <screen>+---------------------+------------------------------------------------+
+| Field               | Value                                          |
++---------------------+------------------------------------------------+
+| admin_state_up      | True                                           |
+| description         |                                                |
+| id                  | 0d973d80-1c79-40a4-881b-42d111ee9625           |
+| listeners           | {"id": "c9a34b63-a1c8-4a57-be22-75264769132d"} |
+|                     | {"id": "4fa2dae0-126b-4eb0-899f-b2b6f5aab461"} |
+| name                | caasp-stack-master_lb-bhr66gtrx3ue             |
+| operating_status    | ONLINE                                         |
+| pools               | {"id": "8c011309-150c-4252-bb04-6550920e0059"} |
+|                     | {"id": "c5f55af7-0a25-4dfa-a088-79e548041929"} |
+| provider            | haproxy                                        |
+| provisioning_status | ACTIVE                                         |
+| tenant_id           | fd7ffc07400642b1b05dbef647deb4c1               |
+| vip_address         | 172.28.0.6                                     |
+| vip_port_id         | 53ad27ba-1ae0-4cd7-b798-c96b53373e8b           |
+| vip_subnet_id       | 87d18a53-ad0c-4d71-b82a-050c229b710a           |
++---------------------+------------------------------------------------+</screen>
+     </step>
+     <step>
+      <para>
+       Search the floating IP list for <literal>vip_address</literal>
+      </para>
+      <screen>&prompt.ardana;openstack floating ip list | grep 172.28.0.6
+| d636f3...481b0c | fd7ff...deb4c1 | 172.28.0.6  | 10.84.65.37  | 53ad2...373e8b |</screen>
+     </step>
+     <step>
+      <para>
+       The load balancer floating ip address is 10.84.65.37
+      </para>
+     </step>
+    </substeps>
+   </step>
+  </procedure>
+  <para>
+   <emphasis role="bold">Accessing the Velum &caasp; Dashboard</emphasis>
+  </para>
+  <para>
+   After the stack has been created, the Velum &caasp; dashboard runs on the
+   admin node. You can access it using the floating IP address of the Admin
+   Node.
+  </para>
+  <para>
+   Create an account and follow the steps in the Velum &caasp; dashboard to
+   complete the &caasp; installation.
+  </para>
+  <para>
+   CaaSP Admin Node Install: Screen 1
```

Comment:
Do these `Admin Nodes` also need to be lowercase too? See all instances below :)

Reply:
Screen titles left in titlecase...matter of opinion I guess. All other admin node case violations fixed.

Thanks for you diligence.